app/static/js/events/scheduler.js

```diff
@@ -919,6 +957,12 @@ function loadMicrolocationsToTimeline(day) {
     $("[data-toggle=tooltip]").tooltip("hide");
 
     if (isReadOnly()) {
+        _.each($microlocations, function ($microlocation) {
+            $microlocation = $($microlocation);
+            if ($microlocation.find('.microlocation-inner').html().trim() === "") {
```

Comment:
The result would be the same ... In most cases ... But by chance (an edge case) due to some error in the template or js code, a non-whitespace character ends up inside the `div`. Then just trimming and equating it to empty space would fail. 

And it's always better to look at the DOM as an object instead of a string.